cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRNonSignedoutMutationDataReader.java

```diff
@@ -164,8 +166,8 @@ public void handleLine(String line) {
         List<MutationRecord> recordsToAnnotate = new ArrayList<>();
         MutationRecord to_add;
         while ((to_add = reader.read()) != null && to_add.getTUMOR_SAMPLE_BARCODE() != null) {
-            // skip if new sample or if mutation record for sample seen already
             if (cvrSampleListUtil.getNewDmpSamples().contains(to_add.getTUMOR_SAMPLE_BARCODE()) ||
+	            !cvrSampleListUtil.getPortalSamples().contains(to_add.getTUMOR_SAMPLE_BARCODE()) ||
```

Comment:
Same comments as in CVRMutationDataReader -   when [cvrSampleListUtil.setNewDmpSamples() is called](https://github.com/knowledgesystems/cmo-pipelines/blob/master/cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/CvrSampleListUtil.java#L120), updatePortalSamples() is called as well....so maybe the additional check that the sample is not in getPortalSamples() is not needed....but it doesn't hurt!

Looks like changes from here to end of class are similar to CVRMutationDataReader.